x-pack/plugins/spaces/public/views/nav_control/components/spaces_menu.tsx

```diff
@@ -77,8 +82,10 @@ export class SpacesMenu extends Component<Props, State> {
     if (items.length === 0) {
       return (
         <EuiText color="subdued" className="eui-textCenter">
-          {' '}
-          no spaces found{' '}
+          <FormattedMessage
+            id="xpack.spaces.view.navControl.spacesMenu.noSpacesFoundTitle"
+            defaultMessage=" no spaces found "
```

Comment:
Please check that spaces you added around `no spaces found` are displayed. If no, use `{' '}` like it was done before

Reply:
@tibmt Has this been checked?